src/Calculator/Views/MainPage.xaml.cpp

```diff
@@ -171,6 +173,7 @@ void MainPage::OnAppPropertyChanged(_In_ Platform::Object ^ sender, _In_ Windows
         else if (newValue == ViewMode::Graphing)
         {
             EnsureGraphingCalculator();
+            KeyboardShortcutManager::HonorShortcuts(false);
```

Comment:
It looks like this can get toggled back on in EquationTextBox_LostFocus and OnNavPaneClosed

Reply:
Good catch, thanks.